postman/Prevention.postman_collection.json

```diff
@@ -0,0 +1,332 @@
+					"options": {
+						"raw": {
+							"language": "json"
+						}
+					}
+				},
+				"url": {
+					"raw": "{{LOCAL_API_URL}}/projects/{{projectGuid}}",
+					"host": [
+						"{{LOCAL_API_URL}}"
+					],
+					"path": [
+						"projects",
+						"{{projectGuid}}"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "GET Parks Section Codes",
+			"request": {
+				"auth": {
+					"type": "bearer",
+					"bearer": [
+						{
+							"key": "token",
+							"value": "{{accessToken}}",
+							"type": "string"
+						}
+					]
+				},
+				"method": "GET",
+				"header": [],
+				"url": {
+					"raw": "{{LOCAL_API_URL}}/codes/bcParksSectionCodes",
+					"host": [
+						"{{LOCAL_API_URL}}"
+					],
+					"path": [
+						"codes",
+						"bcParksSectionCodes"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "GET Parks Section Code",
+			"request": {
+				"auth": {
+					"type": "bearer",
+					"bearer": [
+						{
+							"key": "token",
+							"value": "{{accessToken}}",
+							"type": "string"
+						}
+					]
+				},
+				"method": "GET",
+				"header": [],
+				"url": {
+					"raw": "{{LOCAL_API_URL}}/codes/bcParksSectionCodes/10",
+					"host": [
+						"{{LOCAL_API_URL}}"
+					],
+					"path": [
+						"codes",
+						"bcParksSectionCodes",
+						"10"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Get Project Copy",
+			"request": {
+				"auth": {
+					"type": "bearer",
+					"bearer": [
+						{
+							"key": "token",
+							"value": "{{accessToken}}",
+							"type": "string"
+						}
+					]
+				},
+				"method": "DELETE",
+				"header": [],
+				"url": {
+					"raw": "{{LOCAL_API_URL}}/projects/{{projectGuid}}",
+					"host": [
+						"{{LOCAL_API_URL}}"
+					],
+					"path": [
+						"projects",
+						"{{projectGuid}}"
+					]
+				}
+			},
+			"response": []
+		}
+	]
+}
```

Comment:
Great idea to add the Postman collection. Maybe updating it as we develop can be part of our PR/code review process going forward? Makes it really easy for future developers to pick up API work.

Reply:
It would definitely be great to continue expanding on this.